components/bms_worker/src/CAN/CANIO-tx.c

```diff
@@ -99,78 +106,68 @@ static const packTable_S MSG_packTable_1Hz[] = {
  */
 void CANTX_BUS_A_SWI(void)
```

Comment:
why is there only one SWI? Doing it this way means that your higher priority messages (100Hz in this case) can be slowed down by your low priority messages (1Hz).

Reply:
To be discussed with @JoshLafleur